assets/docs/sources/index.md

```
@@ -6,35 +6,78 @@
 > jmcomic库封装了一套可用于爬取禁漫的Python API.
 >
 > 你可以通过简单的几行Python代码，访问禁漫的接口，以及下载禁漫的本子。
-> 
+>
 > [查看项目更新计划](TODO.md)
 
+## 一图看懂 JMComic 工具生态
```

markdownlint-cli2 (0.18.1)

Heading style
Expected: setext; Actual: atx

(MD003, heading-style)